src/types/verification.ts

```diff
@@ -48,6 +52,10 @@ export interface IVerificationSchema
    * Extracts non-sensitive fields from a transaction
    */
   getPublicView(): PublicTransaction
+  /**
+   * Retrieves payment contact field in a transaction, or undefined if not found
+   */
+  getPaymentContactField(): IVerificationFieldSchema | undefined
```

Comment:
Hm I can definitely get behind specifying `Field` as `FormField`, but as for reducing `PaymentContactField` to `PaymentField`, I'm not sure if it will be confusing in the future as there `PaymentField` more intuitively suggests that it will include contents like the dollar amount, description, etc. when it will just hold the email / future alternate contact information haha

Reply:
yeah, `PaymentContactField` works fine. should we update the model and all related functions with the new terminology? then we stick to `FormFields` and `PaymentContactField` ba